packages/cli/test/unit/commands/domains/ls.test.ts

```diff
@@ -11,16 +11,17 @@ describe('domains ls', () => {
     useDomains();
     client.setArgv('domains', 'ls');
     let exitCodePromise = domains(client);
```

Comment:
Is there a reason we don't just call:
```
const exitCode = await domains(client);
```
here?

Reply:
I was still working through. I'll make this ready for review when I'm done.